### PR TITLE
Improve and fix numerous bugs in stdlib.

### PR DESCRIPTION
**Pull request by Arignir**

### CODE REVIEW

File: stdlib/package.py [hunk 1]
```diff
@@ -293,15 +293,16 @@ def move(self, srcs: str, dst: str):
         :param dst: A path pointing to the destination folder, relative to the ``wrap_cache`` of this :py:class:`.Package`.
         """
 
-        if os.path.isabs(srcs) or os.path.isabs(dst):
+        if os.path.isabs(dst):
             raise RuntimeError("Package.move() received an absolute path as parameter, but it expects a relative one")
```
Comment (doom) [inline stdlib/package.py]: I know this is not introduced by the current commit, but shouldn't you raise `ValueError` instead ?

File: stdlib/manifest.py [hunk 2]
```diff
@@ -179,6 +179,8 @@ def __init__(
         self.instructions = instructions
         self.path = path
 
+        assert(os.path.isabs(self.path))
```
Comment (doom) [inline stdlib/manifest.py]: I think you should raise `ValueError` here instead

File: stdlib/package.py [hunk 3]
```diff
@@ -293,15 +293,16 @@ def move(self, srcs: str, dst: str):
...
 
-        if not os.path.isdir(dst):
-            raise RuntimeError("Package.move() did not receive a path pointing to a directory as paramter")
-
         with stdlib.pushd(self.wrap_cache):
             for srcs in braceexpand.braceexpand(srcs):  # Expand braces
                 for src in glob.glob(srcs):  # Expand globbing
+
+                    if os.path.isabs(src):
+                        raise RuntimeError("Package.move() received an absolute path as parameter, but it expects a relative one")
```
Comment (doom) [inline stdlib/package.py]: Same here about the exception type

File: stdlib/package.py (hunk 1 above)
Comment (Arignir) [reply stdlib/package.py]: I probably should, haven't spent much time looking at all possible errors in Python

**Review by doom — APPROVED** [review]

**Pull request merged**